L1Trigger/L1TTrackMatch/plugins/L1TkEmParticleProducer.cc

```diff
@@ -266,7 +266,7 @@ float L1TkEmParticleProducer::CorrectedEta(float eta, float zv) const {
   } else {
     if (theta > 0)
       delta = ZEcal;
-    if (theta < 0)
```

Comment:
A few lines above `theta` is forced to be positive.
Therefore, the delta for the negative endcap will never be set correctly
This is a bug, which is **not** fixed by this change (it only avoids the static analyzer to complain because tha case `theta=0` would let `delta` uninitialized in the following)

Reply:
(Perhaps just using `eta` instead of `theta` in the `if` clause may be enough to fix. Still, the whole code, at least for this method, is written in a quite inefficient way)

Reply:
@rekovic I believe you are the original author of this code? Can you check this comment?

Reply:
> @rekovic @cecilecaillol Do you agree on the proposed change?

Since @cecilecaillol has already signed this PR, I would say yes...